src/BlazorWasmSdk/Tasks/ComputeBlazorBuildAssets.cs

```diff
@@ -97,7 +103,17 @@ public override bool Execute()
                     }
 
                     var destinationSubPath = candidate.GetMetadata("DestinationSubPath");
-                    if (string.IsNullOrEmpty(destinationSubPath))
+                    if (candidate.GetMetadata("FileName") == "dotnet" && candidate.GetMetadata("Extension") == ".js")
+                    {
+                        var dotNetVersion = BundledNETCoreAppPackageVersion;
+                        var itemHash = FileHasher.GetFileHash(candidate.ItemSpec);
+                        var cacheBustedDotnetJSFileName = $"dotnet.{dotNetVersion}.{itemHash}";
+                        candidate.SetMetadata("RelativePath", $"_framework/{cacheBustedDotnetJSFileName}.js");
+                        candidate.SetMetadata("AssetTraitName", "BlazorWebAssemblyResource");
+                        candidate.SetMetadata("AssetTraitValue", "native");
+                        candidate.SetMetadata("ContentRoot", Path.Combine(IntermediateOutputPath, "blazor"));
```

Comment:
I don't think we are copying this file to `Path.Combine(IntermediateOutputPath, "blazor")` any longer, so I don't think we need the intermediateouputpath here. You should be able to leave it empty and it will receive the content root from [here](https://github.com/dotnet/sdk/blob/main/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets#L211) and will be copied to the output directory from where it will be served